superbuild/projects_modules/openssl.cmake

```diff
@@ -60,8 +60,13 @@ endif(APPLE)
 epComputPath(${ep})
 
 if (WIN32)
```

Comment:
is it normal that we enter here only with WIN32?

Reply:
Set to True when the target system is Windows, including Win64. (extract from https://cmake.org/cmake/help/latest/variable/WIN32.html).